plugin/src/withIosAppcuesRichPush.ts

```diff
@@ -121,19 +140,62 @@ const withAppcuesXcodeProject: ConfigPlugin<ConfigProps> = (config, props) => {
       target.uuid
     );
 
+    const quotedTargetName = `"${APPCUES_NSE_TARGET.NAME}"`;
+
     // Fix "Value for SWIFT_VERSION cannot be empty."
     const swiftVersion = xcodeProject.getBuildProperty('SWIFT_VERSION');
-    xcodeProject.addBuildProperty('SWIFT_VERSION', swiftVersion);
+    xcodeProject.updateBuildProperty(
+      'SWIFT_VERSION',
+      swiftVersion,
+      null, // want both Debug and Release, so don't specify either
+      quotedTargetName
+    );
```

Comment:
The first time around, I couldn't figure out the secret sauce (ie quotes around the target name) to add the build property to only the NSE target, so I used `addBuildProperty`, but `updateBuildProperty` with the quoted target name means we're only changing the target we're creating which is a lot safer, especially with modifying the code signing stuff below.